threads task three.py

```diff
@@ -0,0 +1,45 @@
+"""
+Usage of queues
+"""
+import queue
+import threading
+
+queue = queue.Queue()
+
+
+def put_queue():
+    """
+    The method get inputs and put it in a queue in an infinite loop
+    """
+    while True:
+        inp = input("please input: ")
+        queue.put(inp)
+        if inp == "":
+            break
```

Comment:
`break` is considered a bad habit because it's an unexpected logic break. You should use something more elegant.